docs/use/claim-cfg-rewards/index.md

```diff
@@ -0,0 +1,129 @@
+---
+id: claim-cfg-rewards
+order: 1
+title: Claim CFG rewards
+contributors: <Dennis Wellmann:dennis@centrifuge.io>, <Nico Dreher:nico@centrifuge.io>
+---
+
+# Claiming CFG Rewards
```

Comment:
This needs to be removed for the second level headers (##) to be shown in the Nav bar